src/main/java/org/gridsuite/study/server/service/NetworkModificationTreeService.java

```diff
@@ -959,6 +1029,18 @@ public long countBuiltNodes(UUID studyUuid, UUID rootNetworkUuid) {
         return nodes.stream().filter(n -> self.getNodeBuildStatus(n.getIdNode(), rootNetworkUuid).isBuilt()).count();
     }
 
+    public void invalidateAllChildrenIndexedModifications(UUID parentNodeUuid, UUID networkUuid, boolean includeParentNode, InvalidateNodeInfos invalidateNodeInfos) {
```

Comment:
Rename ?
This method only populate `InvalidateNodeInfos` ?

Reply:
fixed